tests/unit/UrlTest.js

```diff
@@ -224,8 +9,8 @@ describe('Url', () => {
             it('It should work correctly with www in both urls', () => {
                 expect(Url.hasSameExpensifyOrigin('https://www.new.expensify.com/inbox/124', 'https://www.new.expensify.com/action/123')).toBe(true);
             });
-            it('It should work correctly without https://', () => {
-                expect(Url.hasSameExpensifyOrigin('new.expensify.com/action/1234', 'new.expensify.com/action/123')).toBe(true);
```

Comment:
Why this case was removed? It should still work without `https://`

Reply:
URL href attributes always need to have a protocol in them otherwise it is an invalid URL.If you type google.com in any chat on STG and then hover over it to view the href it will show https://google.com, so our logic already creates the correct href

Any url without the protocol and // is treated as a relative URL and is not spec complaint.You can go to any browser and type new URL('google.com') in the console section and it will throw an error.

Therefore, the previous test case was invalid.